### PR TITLE
Feature/map 199/create frontend part for report location

### PR DESCRIPTION
**Pull request by Yojuks**

### CODE REVIEW

File: frontend/src/components/PointPopup/IconsComponent/IconsComponent.tsx [hunk 1]
```diff
@@ -59,6 +65,18 @@ export const IconsComponent: FC<Props> = ({
     setAnchorEl(null);
   };
 
+  // const [report, setReport] = React.useState<any>(false);
+
+  const reportLocation = () => {
+    // setReport({ reported: true })
```
Comment (Yojuks) [inline frontend/src/components/PointPopup/IconsComponent/IconsComponent.tsx:71]: I need advice - how wait for the result setReport?

**Review by naberalgamma — COMMENTED** [review]
if location is my i see 2 report button
image

**Review by naberalgamma — COMMENTED** [review]
first report doing nothing second show Location successfuly edited 
image

**Comment by Yojuks** [comment]
> if location is my i see 2 report button

because this button for moderator and admin

**Comment by naberalgamma** [comment]
i'm just user

**Comment by naberalgamma** [comment]
> i'm just user

**Review by naberalgamma — APPROVED** [review]

**Review by vasylneshchadimov — APPROVED** [review]

**Pull request merged**